packages/web-components/src/popover/popover.styles.ts

```diff
@@ -0,0 +1,31 @@
+import { css } from '@microsoft/fast-element';
+import * as tokens from '../theme/design-tokens';
```

Comment:
Is this going to enable tree shaking? I'm a bit concerned that if I only want to bring in a single component, I'm going to end up with _all_ the tokens.